core/src/main/java/jenkins/management/NodesLink.java

```diff
@@ -55,7 +55,7 @@ public String getDescription() {
     @NonNull
     @Override
     public Permission getRequiredPermission() {
-        return Jenkins.MANAGE;
+        return Jenkins.READ;
```

Comment:
Weird, that looks like an oversight in your initial PR; CLI and Nodes links can be shown to anyone.

Reply:
initial PR? this is the management link from /manage, I haven't touched nodes / agents for system read before this PR

Reply:
IOW, you should change https://github.com/jenkinsci/jenkins/blob/6fb8b93a157e00ac4c057013717e2fef6df1dd09/core/src/main/java/jenkins/management/CliLink.java#L58 😃